README.md

```diff
@@ -0,0 +1,140 @@
+# Child Mind Institute - MindLogger Mobile App
+
+This repository is used for the respondent mobile app of the [MindLogger](https://mindlogger.org/) application stack.
+
+## Application Stack
+
+* MindLogger Admin - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-admin)
+* MindLogger Backend - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-backend-refactor)
+* MindLogger Mobile App - **This Repo**
+* MindLogger Web App - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-web-refactor)
+
+## Getting Started
+
+Running the app:
+
+### 1. Prerequisites
+
+- NodeJS `18` or higher, recommend using `asdf` or `nvm` to manage local node version
```

Comment:
I had problems running this repo with 18+ and `.node-version` mentions node `16`.
Are you sure it runs ok? If so, can we update `.node-version` to match the other repos

Reply:
Yup, I've been running Node 18 and haven't had any issues! Makes sense though to update `.node-version`, good eye!